src/components/ui/Animation.js

```diff
@@ -275,12 +266,14 @@ CIQ.Animation = function (stx, animationParameters, easeMachine) {
             if (this.chart.yAxis.left > x &&
                 this.chart.yAxis.top <= y &&
                 this.chart.yAxis.bottom >= y) {
-                if (flashingColorIndex >= flashingColors.length) { flashingColorIndex = 0; }
-                context.beginPath();
-                context.moveTo(x, y);
-                context.arc(x, y, 2 + flashingColorIndex * 1.07, 0, Math.PI * 2, false);
-                context.fillStyle = flashingColors[flashingColorIndex];
-                context.fill();
+                if (!currentSpotElement) {
+                    currentSpotElement = document.createElement('span');
+                    currentSpotElement.className = 'cq-marker-spot';
+                    this.chart.panel.holder.parentElement.appendChild(currentSpotElement);
+                } else {
+                    currentSpotElement.style.top = `${y - 3}px`;
+                    currentSpotElement.style.left = `${x - 3}px`;
```

Comment:
you need to set `{display: none}` for this when the spot is out of range or when the chart type doesn't support spot.